desc/compute/utils.py

```diff
@@ -188,17 +213,19 @@ def _get_derivs_1_key(key):
     return {key: np.unique(val, axis=0).tolist() for key, val in derivs.items()}
 
 
-def get_profiles(keys, eq=None, grid=None, **kwargs):
+def get_profiles(keys, has_axis, eq=None, grid=None, **kwargs):
```

Comment:
For this and the similar `get_` stuff, could we move the `has_axis` arg to the end and default to `False`? I think the most common use case is when a grid is passed in, so the `has_axis` in that case is redundant, so this would save some code and keep the API the same in most cases.